src/main/java/com/iexec/worker/tee/scone/SconeConfig.java

```diff
@@ -0,0 +1,47 @@
+package com.iexec.worker.tee.scone;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class SconeConfig {
+
+    String sconeCasAddress;
+    String sconeLasAddress;
+    String sconeConfigId;
+
+    @Builder.Default String sconeHeap = "1G";
+    @Builder.Default int sconeLog = 7;
```

Comment:
What is this value for?

Reply:
Those are default value needed to launch scone container.